src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -414,6 +414,9 @@ public int Main(string[] args)
                 var runtimeFolder = c.Option("--runtime",
                     "The folder containing all available framework assemblies.",
                     CommandOptionType.SingleValue);
+                var hideDependent = c.Option("--hide-dependent",
+                    "Hide the dependents",
```

Comment:
It's not the dependencies, it's indeed __dependent__ which introduce the underline packages.